projects/demo/src/modules/services/notifications/notifications.template.html

```diff
@@ -57,21 +57,8 @@
         </button>
         <div i18n class="b-full-width">
             <p>To show notification, use method</p>
-            <p>
-                <code>
-                    <div>show&lt;O, I&gt;(</div>
-                    <div>
-                        &nbsp;&nbsp;&nbsp;&nbsp;content:
-                        PolymorpheusContent&lt;TuiNotificationContentContext&lt;O,
-                        I&gt;&gt;
-                    </div>
-                    <div>
-                        &nbsp;&nbsp;&nbsp;&nbsp;options: TuiNotificationOptions
-                        | TuiNotificationOptionsWithData&lt;I&gt;
-                    </div>
-                    <div>): Observable&lt;O&gt;</div>
-                </code>
-            </p>
+            <tui-doc-code [code]="method"></tui-doc-code>
+            <br />
```

Comment:
https://github.com/Tinkoff/taiga-ui/pull/1184#discussion_r786488101

Reply:
use class `tui-space_bottom-*` if you want to make empty space below

Reply:
fixed